### PR TITLE
Feature/pcs compliance year

### PR DESCRIPTION
**Pull request by JohnPadgettCivica**

### CODE REVIEW

**Comment by JohnPadgettCivica** [comment]
for https://dev.azure.com/civica-digital/EA%20-%20WEEE/_workitems/edit/44475

Ignore the changes to the EvidenceNote date validation attributes im going to remove the weee client and put the test date in the cache.

Still of tests to write,

ListOfNotesViewModelBaseTests
GetEvidenceNotesByOrganisationRequest
Test transfer objects have the correct values being passed
Evidence note data access
ListOfNotesViewModelBaseTests
Redirect tab tests
DateValidation tests

File: src/EA.Weee.Web/Areas/Scheme/ViewModels/ManageEvidenceNotes/ManageEvidenceNoteSchemeViewModel.cs [hunk 1]
```diff
@@ -5,7 +5,7 @@
     using Aatf.ViewModels;
     using Web.ViewModels.Shared;
 
-    public abstract class ManageEvidenceNoteViewModel : IEvidenceNoteRowViewModel
+    public abstract class ManageManageEvidenceNoteSchemeViewModel : IManageEvidenceViewModel
```
Comment (NateHogan) [inline src/EA.Weee.Web/Areas/Scheme/ViewModels/ManageEvidenceNotes/ManageEvidenceNoteSchemeViewModel.cs:8]: ManageManage?
Typo or just the quirk of naming convention?

File: src/EA.Weee.Web.Tests.Unit/ViewModels/ListOfNotesViewModelBaseTests.cs [hunk 2]
```diff
@@ -44,14 +44,14 @@ public void ListOfNotesViewModelBase_ShouldBeAbstract()
         public void ListOfNotesViewModelBase_ShouldHaveIEvidenceNoteRowViewModelAsT()
```
Comment (NateHogan) [inline src/EA.Weee.Web.Tests.Unit/ViewModels/ListOfNotesViewModelBaseTests.cs:44]: Rename test to reflect change

File: src/EA.Weee.Web/Areas/Aatf/ViewModels/FilterViewModel.cs [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ﻿namespace EA.Weee.Web.Areas.Aatf.ViewModels
 {
+    using System;
```
Comment (NateHogan) [inline src/EA.Weee.Web/Areas/Aatf/ViewModels/FilterViewModel.cs:3]: I think this is unused

**Review by NateHogan — COMMENTED** [review]
Seems good, was pretty hard to give a super in-depth view due to size, but outside the minor comments I think it's fine

File: src/EA.Weee.Web/Areas/Scheme/ViewModels/ManageEvidenceNotes/ManageEvidenceNoteSchemeViewModel.cs (hunk 1 above)
Comment (JohnPadgettCivica) [reply src/EA.Weee.Web/Areas/Scheme/ViewModels/ManageEvidenceNotes/ManageEvidenceNoteSchemeViewModel.cs]: Updated...

File: src/EA.Weee.Web.Tests.Unit/ViewModels/ListOfNotesViewModelBaseTests.cs (hunk 2 above)
Comment (JohnPadgettCivica) [reply src/EA.Weee.Web.Tests.Unit/ViewModels/ListOfNotesViewModelBaseTests.cs]: updated...

File: src/EA.Weee.Web/Areas/Aatf/ViewModels/FilterViewModel.cs (hunk 3 above)
Comment (JohnPadgettCivica) [reply src/EA.Weee.Web/Areas/Aatf/ViewModels/FilterViewModel.cs:3]: Will tidy this up after Megs changes come in...

**Pull request merged**